e2e/helpers/dump_browser_logs_helper.js

```diff
@@ -28,7 +28,8 @@ function stringify(value) {
 
 module.exports = class HooksHelpers extends Helper {
   async _failed(test) {
-    const logs = (await this.helpers['Puppeteer'].grabBrowserLogs()).map(log => {
+    const helper = this.helpers['Puppeteer'] || this.helpers['WebDriver'];
```

Comment:
Minor thing. This line repeats a lot in the file so maybe refactor it in a common function?

Reply:
great idea, done.

Reply:
I've added a function in browser_helper.js where there were multiple repeating lines